core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreWebViewClient.java

```diff
@@ -123,7 +121,6 @@ public void onPageFinished(WebView view, String url) {
     Log.d(TAG_KIWIX, "invalidUrl = " + invalidUrl);
```

Comment:
oh, this is never true as it is looking for our legacy prefix, can you update this check to use `ZimFileReader.CONTENT_PREFIX`